bokehjs/src/coffee/models/plots/plot_canvas.coffee

```diff
@@ -69,6 +63,14 @@ class PlotCanvasView extends Renderer.View
     super(options)
     @pause()
 
+    # TODO (bev) this sucks a bit
```

Comment:
Can you elaborate on this comment - so we know what to try and clean-up in the future?